Update bluetooth article for M50.

beaufortfrancois opened the pull request:
This patch will update web bluetooth article when M50 reaches stable channel.

R=@jyasskin @g-ortuno @scheib

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -58,17 +58,22 @@ Bluetooth characteristics.
-but to deploy it on a site you'll need to have HTTPS setup
-on your server. I personally enjoy [GitHub Pages](https://pages.github.com) for
-demo purposes.
+> We care deeply about security, so you will notice that all new Web APIs
```
This isn't quite true; maybe "new web capabilities"? This paragraph also isn't a quote from anywhere so it's a little odd to format it as a blockquote, but I don't care that much about styling.

beaufortfrancois replied:
Thanks! I'll use "new web capabilities" instead of "all new Web APIs".
I use this quote because I wanted to emphasize.

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -58,17 +58,22 @@ Bluetooth characteristics.
+Because this experimental API is a powerful new feature added to the Web,
 Google Chrome aims to make it available only to [secure
-contexts](https://w3c.github.io/webappsec/specs/powerfulfeatures/#intro).  This
+contexts](https://w3c.github.io/webappsec/specs/powerfulfeatures/#intro). This
```
https://w3c.github.io/webappsec-secure-contexts/#intro

beaufortfrancois replied:
Thanks!

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -58,17 +58,22 @@ Bluetooth characteristics.
+need to have HTTPS set up on your server. I personally enjoy [GitHub
+Pages](https://pages.github.com) for demo purposes.
+
 To add HTTPS to your server you'll need to get a TLS certificate and set
```
Maybe mention that certificates are free now?

beaufortfrancois replied:
Done.

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -58,17 +58,22 @@ Bluetooth characteristics.
+Pages](https://pages.github.com) for demo purposes.
+
 To add HTTPS to your server you'll need to get a TLS certificate and set
 it up. Be sure to check out [Security with HTTPS
```
"check out **the** Security with HTTPS article"

beaufortfrancois replied:
Done.

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -100,9 +105,7 @@ Bluetooth 4.0 or later.
-device chooser where he can pick one device or simply cancel the request. At
-the time of writing though, the device chooser hasn't been implemented yet in
-Chrome OS. Only the first device that matches filters will be returned.
+device chooser where he can pick one device or simply cancel the request.
```
s/he/she/ or s/he/they/

beaufortfrancois replied:
Damn! 
I'm trying to be careful with those but I've missed this one ;(

Thank you.

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -198,7 +214,7 @@ full Bluetooth UUID or a short 16- or 32-bit form to `service.getCharacteristic`
 Writing to a Bluetooth GATT Characteristic is as easy as reading it. This time,
 let's reset the value of the Energy Expended field in the Heart Rate
-Measurement characteristic to 0 on a heart rate monitor device.
+Control Point characteristic to 0 on a heart rate monitor device.
```
The Energy Expended field does live in the [Heart Rate Measurement](https://developer.bluetooth.org/gatt/characteristics/Pages/CharacteristicViewer.aspx?u=org.bluetooth.characteristic.heart_rate_measurement.xml); it's just that you use the Control Point to reset it. You could write "let's use the Heart Rate Control Point to reset the value of the Energy Expended field to 0."

beaufortfrancois replied:
Done.

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -220,18 +236,55 @@ navigator.bluetooth.requestDevice({ filters: [{ services: ['heart_rate'] }] })
+{% endhighlight %}
+
+The dedicated [sample](https://googlechrome.github.io/samples/web-bluetooth/notifications.html)
+will show you to stop notifications with `stopNotifications()` and properly
```
"show you **how to** stop notifications"?

beaufortfrancois replied:
Typo. Done.

jyasskin commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -270,9 +312,9 @@ peek of what to expect in the coming months:
   added to improve user experience.
 
-At the time of writing, Chrome OS and Android 6+ are [the most advanced
+At the time of writing, Chrome OS, Android 6+ and Chrome for Linux are [the most advanced
```
Do we say somewhere that Linux requires a very up-to-date kernel and BlueZ?

beaufortfrancois replied:
Yes in https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md
I didn't want to re-add it there to avoid duplication.

> Linux: Requires Kernel 3.19+ and BlueZ 5+ installed (by default in Ubuntu 15.10).
Note that Bluetooth daemon needs experimental interfaces feature enabled: sudo /usr/sbin/bluetoothd -E

jyasskin commented:
I've added a bunch of comments. Thanks for the update!

beaufortfrancois commented on src/content/en/updates/posts/2015/07/interact-with-ble-devices-on-the-web.markdown:
```diff
@@ -270,9 +312,9 @@ peek of what to expect in the coming months:
 platforms](https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md).
-Mac OSX is partially working.  Windows 8.1+, Linux, and iOS will be supported
+Mac OSX is partially working.  Windows 8.1+ and iOS will be supported
 as much as feasible by the platforms.
```
I'll leave it there but we might want to investigate this and add this to https://github.com/WebBluetoothCG/web-bluetooth/blob/gh-pages/implementation-status.md

beaufortfrancois commented:
Thanks for reviewing.
I've addressed your feedback at https://github.com/google/WebFundamentals/pull/2778/commits/1c1f0448c928cb382a76771c4a686cbef96940b9

beaufortfrancois commented:
@reillyeon Does this look good to you for our M51 beta?

reillyeon commented:
Looks reasonable to me but I think you meant to ask @jyasskin.

beaufortfrancois commented:
I did. Thanks @reillyeon  ;)

jyasskin commented:
Yep, looks good. Sorry for not re-checking it earlier!

beaufortfrancois commented:
@petele It should be good to merge and push live now.
